vault/logical_system_test.go

```diff
@@ -229,7 +233,10 @@ func TestSystemBackend_mounts(t *testing.T) {
 		if err != nil {
 			t.Fatalf("err: %v", err)
 		}
+		// Skip over this for now
+		delete(conf.(map[string]interface{}), "deprecation_status")
```

Comment:
Named mount endpoints `/mounts/name` currently do not have a `deprecation_status`, unlike the generic `/mounts`. This should probably be updated, rather than skipping the test.